JapaneseFilter/sections/specific.txt

```diff
@@ -40,6 +40,7 @@ hpplus.jp,tv-tokyo.co.jp,financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.j
 !
 !
 jump.x0.to###prs
+bbs.south-plus.org###header > div[style^="width: 920px"]
```

Comment:
First, we can remove it by blocking the request
`||bbs.south-plus.org/images/masha5-1.gif`

Reply:
Second, isn't this a Chinese site?